tests/test-plugin.c

```diff
@@ -173,8 +173,8 @@ static void test_nonexistent_plugins(void const *test_data)
         l_queue_destroy(plugins_list, NULL);
 
         /*
-          Force plugin name-to-ops lookup to fail excercise additional
-          error paths.
+          Exercise additional error paths by forcing plugin
+          name-to-ops lookup to fail.
```

Comment:
This rewording change inadvertently made it into this pull request.  Should we leave it in place or move it to a different pull request?